babel.config.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// babelrc doesn't respect NODE_PATH anymore but using require does.
+// Alternative to install them locally in node_modules
+module.exports = function (api) {
+  // ensure env is test so that this config won't impact build or dev server
+  if (api.env('test')) {
```

Comment:
i didn't check but is babel config not needed for either build or dev?

Reply:
Checking

Reply:
This is likely used only in a testing environment (Jest) to ensure that the Babel configuration for tests is applied. I guess for bootstrap/build OSD scripts takes care of it.